cluster/gce/gci/configure-helper.sh

```diff
@@ -66,11 +68,65 @@ function create-dirs {
   fi
 }
 
-# Formats the given device ($1) if needed and mounts it at given mount point
+# Gets the total number of $(1) and $(2) type disks specified
+# by the user in ${NODE_LOCAL_SSDS_EXT}
+function get-local-disk-num() {
+  local interface=$1
+  local format=$2
+
+  total=0
+  if [[ ! -z ${NODE_LOCAL_SSDS_EXT} ]]; then
+    IFS=";" read -r -a ssdgroups <<< ${NODE_LOCAL_SSDS_EXT}
+    for ssdgroup in "${ssdgroups[@]}"
+    do
+      IFS="," read -r -a ssdopts <<< ${ssdgroup}
+      local opnum=${ssdopts[0]}
+      local opinterface=${ssdopts[1]}
+      local opformat=${ssdopts[2]}
+
+      if [[ ${opformat,,} == ${format,,} && ${opinterface,,} == ${interface,,} ]]; then
+        total=$((total+opnum))
+      fi
+    done
+  fi
+}
+
+# Creates a symlink for a ($1) so that it may be used as block storage
+function safe-block-symlink(){
+  local device=$1
+  local interface=$2
+  
+  mkdir -p "${UUID_MNT_PREFIX}-${interface}-block"
+  ssdmap="/home/kubernetes/localssdmap.txt"
+  echo "Creating symlink for block devices..."
+
+  if [[ ! -e ${ssdmap} ]]; then
+    touch ${ssdmap}
+    chmod +w ${ssdmap}
+  fi
+
+  # each line looks like "${device} persistent-uuid"
+  if [[ ! -z $(cat ${ssdmap} | grep ${device}) ]]; then
```

Comment:
You could just do grep $device $ssdmap.

I would add a space after device, the handle the case where you have 2 devices and the first is a substring of the second.